simulation/src/test/resources/report.md

```diff
@@ -2,35 +2,35 @@
 one_endpoint_dies_on_each_server[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=64.4%	client_mean=PT2.6866096S   	server_cpu=PT25M          	client_received=2500/2500	server_resps=2500	codes={200=1611, 500=889}
- one_endpoint_dies_on_each_server[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=65.2%	client_mean=PT1.1283168S   	server_cpu=PT25M          	client_received=2500/2500	server_resps=2500	codes={200=1629, 500=871}
-           one_endpoint_dies_on_each_server[UNLIMITED_ROUND_ROBIN].txt:	success=63.9%	client_mean=PT0.6S         	server_cpu=PT25M          	client_received=2500/2500	server_resps=2500	codes={200=1598, 500=902}
+ one_endpoint_dies_on_each_server[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=65.9%	client_mean=PT1.0230944S   	server_cpu=PT25M          	client_received=2500/2500	server_resps=2500	codes={200=1648, 500=852}
+           one_endpoint_dies_on_each_server[UNLIMITED_ROUND_ROBIN].txt:	success=64.8%	client_mean=PT0.6S         	server_cpu=PT25M          	client_received=2500/2500	server_resps=2500	codes={200=1620, 500=880}
        simplest_possible_case[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=100.0%	client_mean=PT0.799924242S 	server_cpu=PT2H55M59S     	client_received=13200/13200	server_resps=13200	codes={200=13200}
-           simplest_possible_case[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT0.785727272S 	server_cpu=PT2H52M51.6S   	client_received=13200/13200	server_resps=13200	codes={200=13200}
-                     simplest_possible_case[UNLIMITED_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT0.785727272S 	server_cpu=PT2H52M51.6S   	client_received=13200/13200	server_resps=13200	codes={200=13200}
+           simplest_possible_case[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT0.784590909S 	server_cpu=PT2H52M36.6S   	client_received=13200/13200	server_resps=13200	codes={200=13200}
+                     simplest_possible_case[UNLIMITED_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT0.784590909S 	server_cpu=PT2H52M36.6S   	client_received=13200/13200	server_resps=13200	codes={200=13200}
         slow_503s_then_revert[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=100.0%	client_mean=PT0.137438962S 	server_cpu=PT6M39.829999979S	client_received=3000/3000	server_resps=3077	codes={200=3000}
-            slow_503s_then_revert[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT0.093312988S 	server_cpu=PT4M35.008666652S	client_received=3000/3000	server_resps=3039	codes={200=3000}
-                      slow_503s_then_revert[UNLIMITED_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT0.093312988S 	server_cpu=PT4M35.008666652S	client_received=3000/3000	server_resps=3039	codes={200=3000}
+            slow_503s_then_revert[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT0.081574996S 	server_cpu=PT4M2.768666661S	client_received=3000/3000	server_resps=3017	codes={200=3000}
```

Comment:
these graphs are also kinda fun to look at, as the kinda illustrate how when a server's performance suddenly changes, this 'running timers' heuristic stops us charging off a cliff at our old request rate (you can see this in the big yellow spike on the left).

https://github.com/palantir/dialogue/blob/dfox/running-timers/simulation/src/test/resources/report.md#slow_503s_then_revertconcurrency_limiter_round_robin